common/core/overlaylayer.h

```diff
@@ -316,6 +324,9 @@ struct OverlayLayer {
   LayerComposition supported_composition_ = kAll;
   LayerComposition actual_composition_ = kAll;
   HWCLayerType type_ = kLayerNormal;
+
+  struct hdr_metadata hdrmetadata;
```

Comment:
property should be named with "_" at the end to be different with variable in method

Reply:
hdr_metadata_
color_space_